src/test/java/com/google/devtools/build/lib/bazel/repository/skylark/SkylarkRepositoryContextTest.java

```diff
@@ -145,16 +145,16 @@ public void testWhich() throws Exception {
   @Test
   public void testFile() throws Exception {
     setUpContexForRule("test");
-    context.createFile(context.path("foobar"), "", true, null);
-    context.createFile(context.path("foo/bar"), "foobar", true, null);
-    context.createFile(context.path("bar/foo/bar"), "", true, null);
+    context.createFile(context.path("foobar"), "", true, true, null);
```

Comment:
These call sites can use some param name comments.

Reply:
Sure. That sounds like a reasonable cleanup that could be applied to this test file in a separate PR, or internal CL.

Reply:
Granted they should've been there in the pre-existing code. I mention it because you're creating an additional parameter which makes the readability slightly worse, but I won't block this PR on it.